### ISSUE TITLE
Create HTML version of Help to make available through website

### ISSUE THREAD

**Issue by ACowan0105** [virtualcell/vcell#186]
Help originally create by Li Ye, can find by checking repository for Li Ye

**Comment by jcschaff** [comment]
The technology for web based help has evolved over time, need to see what is our best option.  Nice thing is that our help system is encoded in a structured/semantic way.  Don't recall what Li Ye did years ago, but probably a good time to consider newer solutions.

**Comment by ACowan0105** [comment]
@jcschaff we have discussed this several times over the years.  At one point Dan found a newer version of the java help we had used, but it never progressed beyond that.

**Comment by ACowan0105** [comment]
Create a way to use the current documentation structure to automatically output an HTML file of the current Help version and deposit within the web site.

**Comment by jcschaff** [comment]
see #1225 

- update 7.5 alpha help prior to release)

see #1239
- refactored help generator to pave way for separating web and desktop help
- improved index.html on web using nested lists
- improved image layout (still not quite right)
- Github Actions now automatically deploy each time software is deployed
- see up-to-date but poorly formated web help [vcell.org web help](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/index.html)

**Comment by jcschaff** [comment]
@AlexPatrie can you suggest small changes to HTML markup or CSS to improve our online help.  Also - could generate markdown or other content which may be more search or LLM friendly.

**Comment by AlexPatrie** [comment]
> @AlexPatrie can you suggest small changes to HTML markup or CSS to improve our online help. Also - could generate markdown or other content which may be more search or LLM friendly.

@jcschaff Yes! I would be happy to take a look. I will start with [this link.](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/index.html)

**Comment by vcellmike** [comment]
New pages are not displayed online, while are shown in TOC. See
[Importing HDF5 files into FIJI/ImageJ(https://vcell.org/webstart/VCell_Tutorials/VCell_Help/topics/ch_3/BioModelApplications/Simulations/Results/Spatial/OpenHDF5ImageJ.html)
[Annotations Panel](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/topics/ch_8/PropertiesPanes/PP_Annotations.html)
[Point Location Process Kinematics Properties Pane](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/topics/ch_8/PropertiesPanes/PP_KinematicsPointLocationProcess.html)
[Point Object Kinematics Properties Pane](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/topics/ch_8/PropertiesPanes/PP_KinematicsPointObject.html)
[Surface Object Kinematics Properties Pane](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/topics/ch_8/PropertiesPanes/PP_KinematicsSurfaceObject.html)
[Surface Process Kinematics Properties Pane](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/topics/ch_8/PropertiesPanes/PP_KinematicsSurfaceProcess.html)
[Volume Object Kinematics Properties Pane](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/topics/ch_8/PropertiesPanes/PP_KinematicsVolumeObject.html)
[Volume Process Kinematics Properties Pane](https://vcell.org/webstart/VCell_Tutorials/VCell_Help/topics/ch_8/PropertiesPanes/PP_KinematicsVolumeProcess.html)

**Comment by AvocadoMoon** [comment]
already created
